src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -291,7 +294,7 @@ class MarathonSchedulerService @Inject() (
     leader.set(true)
     runDriver(abdicateOption)
 
-    schedulerActor ! Start
+    eventStream.publish(LeadershipStateEvent.ElectedAsLeader)
```

Comment:
This is cool for logic, that gets executed sometimes in the future after the leader was elected. I could think of other logic (think of migration etc), that has to be executed after election but before the driver starts. I know this is not part of this PR, but would be really useful.